src/main/java/nl/tudelft/contextproject/controller/GameController.java

```diff
@@ -181,11 +185,13 @@ private Vector2f attachMazeTiles(Level level) {
 		Vector2f start = new Vector2f();
 		for (int x = 0; x < level.getWidth(); x++) {
 			for (int y = 0; y < level.getHeight(); y++) {
+				attachRoofTile(x, y);
```

Comment:
Could be moved below ` if (level.isTileAtPosition(x, y)) {` to only generate a roof when there is a tile below.